SwrveSDK/src/main/java/com/swrve/sdk/localstorage/MemoryCachedLocalStorage.java

```diff
@@ -18,7 +18,8 @@
     private ILocalStorage cache;
     private ILocalStorage secondaryStorage;
 
-    private Object eventLock = new Object();
+    public static final Object EVENT_LOCK = new Object();
```

Comment:
I guess it could be, but tbh none of this is very elegant. Would like to change a few more things like that.